src/Hpack/Config.hs

```diff
@@ -1037,36 +1037,48 @@ toLibrary dir name globalOptions =
       Library librarySectionExposed exposedModules otherModules generatedModules reexportedModules signatures
       where
         (exposedModules, otherModules, generatedModules) =
-          determineModules pathsModule inferableModules (ModuleSpecification librarySectionExposedModules librarySectionGeneratedExposedModules) (ModuleSpecification librarySectionOtherModules librarySectionGeneratedOtherModules)
+          determineModules DetermineModules {
+              _pathsModule = pathsModule
+            , _inferableModules = inferableModules
+            , _exposedModules = librarySectionExposedModules
+            , _generatedExposedModules = librarySectionGeneratedExposedModules
+            , _otherModules = librarySectionOtherModules
+            , _generatedOtherModules = librarySectionGeneratedOtherModules
+          }
```

Comment:
This is what I meant with simulating keyword arguments, not necessarily the right thing to do, just to make the point.

Reply:
@quasicomputational

Reply:
:thumbsup:

Definitely cleaner this way than trying to exploit the parallel between exposed and other modules.